src/main/java/MinesweeperController.java

```diff
@@ -213,12 +217,15 @@ public void revealCell(MinesweeperCell cell) {
             default:
                 cell.setGraphic(null);
         }
+        unrevealedCells--;
     }
 
     private void revealAll() {
         for (MinesweeperCell[]row: board) {
             for (MinesweeperCell cell: row) {
-                revealCell(cell);
+                if (!cell.isDisabled()){
```

Comment:
Is this needed? this is only called when the game is over, don't we want to reveal all cells at that point? @scroitoru